boards/imix/src/io.rs

```diff
@@ -14,17 +14,20 @@ impl Write for Writer {
         let uart = unsafe { &mut sam4l::usart::USART3 };
         if !self.initialized {
             self.initialized = true;
-            uart.configure(sam4l::usart::USARTParams {
+            uart.init(uart::UARTParams {
                 baud_rate: 115200,
-                data_bits: 8,
+                stop_bits: uart::StopBits::One,
                 parity: uart::Parity::None,
-                mode: uart::Mode::Normal,
+                hw_flow_control: false,
             });
+            uart.reset();
             uart.enable_tx();
 
         }
+        // XXX: I'd like to get this working the "right" way, but I'm not sure how
         for c in s.bytes() {
             uart.send_byte(c);
+            while !uart.tx_ready() {}
         }
         Ok(())
     }
```

Comment:
While I was in this file, I noticed that it has `... fn panic_fmt(...` whereas storm/src/io.rs has `... fn rust_begin_unwind(...`. I know the storm one works. Is the imix one correct, @alevy?